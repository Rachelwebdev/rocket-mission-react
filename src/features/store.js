import { configureStore } from '@reduxjs/toolkit';
import missionReducer from './missions/missionSlice';
import rocketsReducer from './rockets/rocketsSlice';

const store = configureStore({
  reducer: {
    missions: missionReducer,
    rockets: rocketsReducer,
  },
});
export default store;
