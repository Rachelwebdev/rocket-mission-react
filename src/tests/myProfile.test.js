import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../features/store';

describe('testing snapshots of Rocket component', () => {
  it('testing myProfile component', () => {
    const profileComponent = render(
      <Provider store={store}>
        <myProfile />
      </Provider>
    );
    expect(profileComponent).toMatchSnapshot();
  });
});
