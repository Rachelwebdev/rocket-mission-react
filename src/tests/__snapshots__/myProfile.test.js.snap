// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Profile Page components testing ProfilePage component 1`] = `
<div
  className="mainProfile"
>
  <div
    className="container"
  >
    <h1>
      Reserved Rockets
    </h1>
    <ul />
  </div>
  <div
    className="container"
  >
    <h1>
      Reserved Missions
    </h1>
    <ul />
  </div>
</div>
`;
